src/base/breadcrumb/_print.scss

```diff
@@ -0,0 +1,20 @@
+/*
+  WET-BOEW
+  @title: Breadcrumb print CSS overrides
+ */
+
+@mixin breadcrumb-print {
```

Comment:
If you want to conditionally include it, maybe another variable "$wb-bc-print-enable"?